core/modules/system/config/system.core.json

```diff
@@ -76,6 +78,7 @@
     "site_slogan": "",
     "theme_default": "stark",
     "theme_debug": 0,
+    "use_cache_tags_cache": 1,
```

Comment:
Maybe these new variables should be disabled by default and then updated in the update hook. Or disable them in `system_update_1000()` temporarily.

Reply:
Scratch that. I was mistaken: I thought that `use_cache_tags_cache` would toggle the whole cache tags thing.